typescript/s3-kms-cross-account-replication/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "s3-encrypted-cross-account-replication-cdk",
+  "version": "0.1.0",
+  "bin": {
+    "s3-encrypted-cross-account-replication-cdk": "bin/s3-encrypted-cross-account-replication-cdk.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@types/jest": "^26.0.10",
+    "@types/node": "10.17.27",
+    "aws-cdk": "2.19.0",
+    "jest": "^26.4.2",
+    "ts-jest": "^26.2.0",
+    "ts-node": "^9.0.0",
+    "typescript": "~3.9.7"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-iam": "^1.151.0",
```

Comment:
since you pulled in `aws-cdk-lib`,you don't need to pull the `@aws-cdk/***` libraries

Reply:
Thanks! I removed the 1.x dependencies